daemon-tests/tests/happy_path.rs

```diff
@@ -395,16 +395,32 @@ async fn force_close_open_cfd(maker_position: Position) {
 #[tokio::test]
 async fn rollover_an_open_cfd_maker_going_short() {
     let _guard = init_tracing();
-    rollover_an_open_cfd(Position::Short).await;
+    rollover_an_open_cfd(Position::Short, 1).await;
 }
 
 #[tokio::test]
 async fn rollover_an_open_cfd_maker_going_long() {
     let _guard = init_tracing();
-    rollover_an_open_cfd(Position::Long).await;
+    rollover_an_open_cfd(Position::Long, 1).await;
 }
 
-async fn rollover_an_open_cfd(maker_position: Position) {
+#[tokio::test]
+async fn double_rollover_an_open_cfd_maker_going_short() {
+    // double rollover ensures that both parties properly succeeded can do another rollover
+
+    let _guard = init_tracing();
+    rollover_an_open_cfd(Position::Short, 2).await;
+}
+
+#[tokio::test]
+async fn double_rollover_an_open_cfd_maker_going_long() {
+    // double rollover ensures that both parties properly succeeded can do another rollover
+
+    let _guard = init_tracing();
+    rollover_an_open_cfd(Position::Long, 2).await;
+}
```

Comment:
:question: Why do we need to test this both for long and short? I think we introduced certain taker-goes-short tests to mark the milestone at the time, but I'm not sure there is any major difference at the actor level between long and short. I suspect we don't need to encumber ourselves with duplicated tests in this layer.

Reply:
I added them for having more complete tests. The advantage is marginal - happy to remove one of them.

Reply:
I removed one of the tests, adapted test name and comment.